api/addNotes.ts

```diff
@@ -0,0 +1,27 @@
+import { authHeaders, baseUrl } from "./base";
+import { ListItem } from "./types";
+
+export const addNotes = (notes: string, list: ListItem) => {
+  console.log("list item: ", list);
+  fetch(`${baseUrl()}/api/v1/edit_list_item`, {
+    method: "post",
+    headers: { ...authHeaders() },
+    body: JSON.stringify({
+      id: list.id,
+      book_id: list.book.id,
+      user_id: list.user_id,
+      rating: list.book.rating,
+      notes: notes,
+      start_date: list.book.start_date,
+      finish_date: list.book.finish_date,
+      created_at: list.book.created_at,
+      updated_at: list.book.updated_at,
+    }),
+  }).then((res) => {
+    if (res.ok) {
+      return res.json();
+    } else {
+      return res.json();
+    }

```

Comment:
Are you sure this logic is correct? 

```suggestion
    return res.json();
```

Reply:
Sorry, I removed a function previously in here. I forgot to remove the conditional statement.